pinot-broker/src/main/java/org/apache/pinot/broker/broker/BrokerAdminApiApplication.java

```diff
@@ -100,7 +103,13 @@ public void start(List<ListenerConfig> listenerConfigs) {
     } catch (IOException e) {
       throw new RuntimeException("Failed to start http server", e);
     }
-    PinotReflectionUtils.runWithLock(this::setupSwagger);
+    PinotConfiguration brokerConf = (PinotConfiguration) getProperties().get(PINOT_CONFIGURATION);
```

Comment:
Create a variable in constructor:
`_swaggerBrokerEnabled = brokerConf.getProperty(CommonConstants.Broker.CONFIG_OF_SWAGGER_BROKER_ENABLED, CommonConstants.Broker.DEFAULT_SWAGGER_BROKER_ENABLED)`

Then in start(...) just do 
```
if (_swaggerBrokerEnabled) {
    PinotReflectionUtils.runWithLock(this::setupSwagger);
} else {
    LOGGER.info("Hiding Swagger UI for Broker");
```

Reply:
Fixed. Thanks for the review